webapp/sources/rudder/rudder-web/src/main/webapp/style/rudder/rudder-directives.css

```diff
@@ -43,7 +43,8 @@
   height: calc(100VH - 80px);
 }
 #policyConfiguration,
-#policyConfiguration > form{
+#policyConfiguration > form,
+#policyConfiguration form > form{
```

Comment:
is it impossible to have two forms ??

Reply:
no it's ok to have several forms ! i guess we are not replacing correct element here ...